### PR TITLE
changed some sentences

### PR DESCRIPTION
**Pull request by toger5**

### CODE REVIEW

**Comment by Geequlim** [comment]
Thanks for your help!

File: README.md [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
 * Run `python install.py` to install the themes.
-* Select the theme path end with `.tres` under your install directory in godot editor with `Settings/Global/Custom Theme`.
+* Select the theme path ending with `.tres` under your install directory in the godot editor with `Settings/Global/Custom Theme`.
 
 You can pass a param to run `install.py` to set install directory.
 
 ### Default install directory
 
-It should put into the `theme` folder under your `.godot` folder.
+It should install it into the `theme` folder under your `.godot` folder.
```
Comment (Geequlim) [inline README.md:17]: Double it is this ok?

File: README.md [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
 
-I don't know the where the `.godot` folder located on Windows platform, so I just set it to `C://.godot/theme` you can pull a PR to fix it in `install.py` and remove this message
+I don't know the where the `.godot` folder is located on Windows platform, so I just set it to `C://.godot/theme` you can pull a PR to fix it in `install.py` and remove this message
```
Comment (Geequlim) [inline README.md:18]: And the around `where`

File: README.md (hunk 1 above)
Comment (toger5) [reply README.md:17]: I don't exactly know what you mean? Do you think it is bad that the headline is mentioning: "install" and the the sentence too?

**Comment by Geequlim** [comment]
Sorry about my poor English.
I think the sentences may still containing grammar errors.

File: README.md (hunk 1 above)
Comment (toger5) [reply README.md:17]: Now i got it :)
first it represents the installer the second one the theme

File: README.md (hunk 2 above)
Comment (toger5) [reply README.md:18]: yea you are right. I forgot to delete one "the"

**Comment by toger5** [comment]
Do you want me to go over it again and than make a new PR (where I miss out less stuff :) )?

**Comment by Geequlim** [comment]
Yes, you can update it :)

**Comment by toger5** [comment]
I created a new pr. Just delete this one...

**Pull request closed without merging**